yt/chyt/controller/internal/agent/agent.go

```diff
@@ -492,3 +565,8 @@ func (a *Agent) OperationNamespace() string {
 func (a *Agent) CheckHealth() error {
 	return a.healthState.Get()
 }
+
+type scalingRequest struct {
+	OpletAlias    string
```

Comment:
Let's add opID here to be sure that we won't apply outdated scalingRequest to a new operation after its restart